### PR TITLE
Feature/player selection

### PR DESCRIPTION
**Pull request by m4rtint**
This is for - The arrows having colours on the panels

### CODE REVIEW

**Comment by DJSteins** [comment]
Can you add some summaries/comments?

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarAppearance/HatSlotViewController.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarAppearance/HatSlotViewController.cs:1]: Missing Header

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarAppearance/SkinSlotViewController.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using _PartyParkade.Script.ScriptableObject.Materials;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarAppearance/SkinSlotViewController.cs:1]: Missing Header

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarAppearance/SlotViewController.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
+﻿using _PartyParkade.Script.Data;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarAppearance/SlotViewController.cs:1]: Missing Header

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarShowCase/CarShowCaseView.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
+﻿using System;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarShowCase/CarShowCaseView.cs:1]: Missing Header

**Comment by mikeslgoh** [comment]
Add the autosquash label once you've completed the changes

**Comment by mikeslgoh** [comment]
[Re-work player selection UI](https://trello.com/c/mQM3kHdo/447-re-work-player-selection-ui)

**Pull request merged**